salsah/src/public/js/jquery.htmleditor.js

```diff
@@ -165,7 +165,8 @@
 							
 						}
 					},
-					toolbar: [ ['Source', 'Bold', 'Italic', 'Underline', 'Strike', 'Subscript', 'Superscript', '-','RemoveFormat', 'Link', 'Unlink'] ] // configuration for toolbar buttons, must conform to `STANDARD_MAPPING`
+					toolbar: [ ['Source', 'Bold', 'Italic', 'Underline', 'Strike', 'Subscript', 'Superscript', '-','RemoveFormat', 'Link', 'Unlink'] ], // configuration for toolbar buttons, must conform to `STANDARD_MAPPING`
+					removeButtons: ''
```

Comment:
I see now that on the previous ckEditor, you (?) emptied that file.  
We can also do that.